libavcodec/vaapi_encode.c

```diff
@@ -2520,6 +2533,10 @@ av_cold int ff_vaapi_encode_init(AVCodecContext *avctx)
         }
     }
 
+    ctx->encode_fifo = av_fifo_alloc((MAX_PICTURE_REFERENCES+1)* sizeof(VAAPIEncodePicture *));
```

Comment:
"MAX_PICTURE_REFERENCES+1)*" -> "MAX_PICTURE_REFERENCES + 1) *". You can use tools/patchcheck to scan your patch, it can detect this kind of style issue. Others looks good to me.

Reply:
done